docs/start_guide.md

```diff
@@ -0,0 +1,232 @@
+## Getting Started
+
+A typical Atomic App or "Nulecule" container consists of the following files:
+```sh
+~/helloapache
+▶ tree
+.
+├── answers.conf.sample
+├── artifacts
+│   ├── docker
+│   │   └── hello-apache-pod_run
+│   ├── kubernetes
+│   │   └── hello-apache-pod.json
+│   └── marathon
+│       └── helloapache.json
+├── Dockerfile
+├── Nulecule
+└── README.md
+```
+
+We will go through each file and folder as we build our first Atomic App container.
+
+For this example, we will be using the [helloapache](https://github.com/projectatomic/nulecule-library/tree/master/helloapache) example from the [nulecule-library](https://github.com/projectatomic/nulecule-library) repo.
+
+In order to follow along, fetch the container and `cd` into the directory:
+```sh
+atomicapp fetch --destination helloapache projectatomic/helloapache
+cd helloapache
+```
+
+### ~/helloapache/Dockerfile
+Atomic App itself is packaged as a container. End-users typically do not install the software from source, instead using the `atomicapp` container as the `FROM` line in a Dockerfile and packaging your application on top. For example:
+
+```Dockerfile
+FROM projectatomic/atomicapp
+
+MAINTAINER Your Name <you@example.com>
+
+ADD /Nulecule /Dockerfile README.md /application-entity/
+ADD /artifacts /application-entity/artifacts
+```
+
+Within `helloapache` we specify a bit more within our labels:
+```Dockerfile
+FROM projectatomic/atomicapp:0.4.2
+
+MAINTAINER Red Hat, Inc. <container-tools@redhat.com>
+
+LABEL io.projectatomic.nulecule.providers="kubernetes,docker,marathon" \
+      io.projectatomic.nulecule.specversion="0.0.2"
+
+ADD /Nulecule /Dockerfile README.md /application-entity/
+ADD /artifacts /application-entity/artifacts
+```
+
+__Optionally__ indicating what providers are included within our container:
+```Dockerfile
+LABEL io.projectatomic.nulecule.providers="kubernetes,docker" \
+      io.projectatomic.nulecule.specversion="0.0.2"
+```
```

Comment:
do we need this again or should we just reference the two labels in the above block and say what they are for.